runtime/errors.go

```diff
@@ -152,14 +154,29 @@ func DefaultHTTPError(ctx context.Context, mux *ServeMux, marshaler Marshaler, w
 	}
 
 	handleForwardResponseServerMetadata(w, mux, md)
-	handleForwardResponseTrailerHeader(w, md)
+
+	//RFC 7230 https://tools.ietf.org/html/rfc7230#section-4.1.2
+	//Unless the request includes a TE header field indicating "trailers"
+	//is acceptable, as described in Section 4.3, a server SHOULD NOT
+	//generate trailer fields that it believes are necessary for the user
+	//agent to receive.
```

Comment:
Nit: please add a space after the `//` since that is the convention.

Reply:
Done ;)